packages/backend-core/src/tests/globalTeardown.ts

```diff
@@ -0,0 +1,12 @@
+import { getContainerRuntimeClient } from "testcontainers"
+
+export default async function teardown() {
+  const client = await getContainerRuntimeClient()
+
+  // @ts-expect-error
+  // eslint-disable-next-line no-undef
+  const containerId = globalThis.__COUCHDB_CONTAINER_ID__
+  const container = client.container.getById(containerId)
+  await client.container.stop(container)
```

Comment:
Not sure if there is any real benefit of doing this. The containers will stop automatically, and this might add some await time to the tests for no reason

Reply:
Ah, do they? This is just for the CouchDB container, I didn't realise there was some existing teardown process.

Reply:
AFAIK test containers check if they are attached to a process, otherwise, they kill themselves